.gitignore

```diff
@@ -8,4 +8,7 @@ keys/
 docs/generated
 docs/README.rst
 protos/
```

Comment:
This should be removed from here, it kinda breaks updates from the linked repository.

Reply:
U mean protos?